packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -498,8 +498,8 @@ msgstr ""
 msgid "Browse"
 msgstr "Navegui"
 
-#. Default: "Browse the site, drop an image, or type an URL"
-#: components/manage/Blocks/Image/Edit
+#. Default: "Browse the site, drop an image, or use an URL"
```

Comment:
This needs to be consistent. Some people [pronounce this as /ɝːl/](https://dictionary.cambridge.org/us/pronunciation/english/earl), but the [correct pronunciation is /ˌjuː.ɑːrˈel/](https://dictionary.cambridge.org/us/pronunciation/english/url). [Whether to use the indefinite article "a" or "an" depends on the _sound_ that starts the following word, not the first letter](https://www.merriam-webster.com/grammar/is-it-a-or-an). Thus the correct phrase is "a URL".

I updated this in only one place, so a global find and replace is needed.

```suggestion
#. Default: "Browse the site, drop an image, or use a URL"
```

Reply:
Done

Reply:
Nope. Fixed in [40c2778](https://github.com/plone/volto/pull/5607/commits/40c2778422e845d63ebaf080250aa2fffa8fd93e)